src/CoreBundle/Bridge/Chameleon/Migration/Script/update-1553768043.inc.php

```diff
@@ -0,0 +1,29 @@
+<h1>Build #1553768043</h1>
+<h2>Date: 2019-03-28</h2>
+<div class="changelog">
+    - Reset field class and fix translations for cms_media.width
+</div>
+<?php
+
+$fieldId = TCMSLogChange::GetTableFieldId(TCMSLogChange::GetTableId('cms_media'), 'width');
```

Comment:
Doesn't this field (still) have the wrong type? Properties if I remember correctly. There should be a ticket / PR for this somewhere.

Reply:
The type is correct, it's just the field class that needs to be removed so that the default class is used - this is done below in line 13.